src/containers/PermissionsApplication.jsx

```diff
@@ -57,7 +57,13 @@ const PermissionsApplication = ({ match }) => {
           ).map(([key, value]) => (
             <Link to={`/permissions/${appName}/${key}`} key={key}>
               <Typography variant="h4">
-                {value.type} : {value.verbs ? value.verbs.join(' / ') : 'ALL'}
+                {value.type} :{' '}
+                {!value.verbs ||
+                (value.verbs.length > 1 && value.verbs.indexOf('GET') >= 0)
+                  ? 'Lecture et Écriture'
+                  : value.verbs.length === 1 && value.verbs[0] === 'GET'
+                  ? 'Lecture'
+                  : 'Écriture'}
```

Comment:
is it possible to add 
```
findABetterNamePlease: {
                type: 'io.cozy.apps',
                description:
                  'Required by the cozy-bar to display the icons of the apps',
                verbs: ['WRITE']
              }
              ```
              line 69 of PermissionsApplications, and update snapshot, in order to add 'Ecriture' in the snap?